apps/examples/testcase/le_tc/kernel/tc_libc_timer.c

```diff
@@ -393,35 +406,35 @@ static void tc_libc_timer_strftime(void)
 	strftime(buffer, BUFF_SIZE, "%P", &st_time);
 	TC_ASSERT_EQ("strftime", strcmp(buffer, "am"), 0);
 
+	strftime(buffer, BUFF_SIZE, "%s", &st_time);
+	TC_ASSERT_EQ("strftime", atoi(buffer), mktime((FAR struct tm *)&st_time));
+
 	/* Check the second as a decimal number (range 00 to 60) */
 	strftime(buffer, BUFF_SIZE, "%S", &st_time);
 	TC_ASSERT_EQ("strftime", atoi(buffer), st_time.tm_sec);
 
+	strftime(buffer, BUFF_SIZE, "%t", &st_time);
+	TC_ASSERT_EQ("strftime", strcmp(buffer, "\t"), 0);
+
 	/* Check the year as a decimal number including the century */
 	strftime(buffer, BUFF_SIZE, "%Y", &st_time);
 	TC_ASSERT_EQ("strftime", atoi(buffer), st_time.tm_year + TM_YEAR_BASE);
 
-	/* Check the year as a decimal number without a century (range 00 to 99) */
-	strftime(buffer, BUFF_SIZE, "%y", &st_time);
-	TC_ASSERT_EQ("strftime", atoi(buffer), st_time.tm_year % 100);
-
-	/* Check with invalid param. it will returns 0. */
-	ret_chk = strftime(buffer, BUFF_SIZE, "%f", &st_time);
-	TC_ASSERT_EQ("strftime", ret_chk, 0);
-	TC_ASSERT_EQ("strftime", atoi(buffer), 0);
+	strftime(buffer, BUFF_SIZE, "%%", &st_time);
+	TC_ASSERT_EQ("strftime", strcmp(buffer, "%"), 0);
 
 	TC_SUCCESS_RESULT();
 }
 
 /**
-* @fn             :tc_libc_timer_strptime
-* @brief          :strptime - Format string as time
-* @Scenario       :strptime - converts time string as the time described in timeptr
-*                  structure as per the format specifiers.
-* API's covered   :strptime
-* Preconditions   :none
-* Postconditions  :none
-* @return         :void
+* @fn				:tc_libc_timer_strptime
+* @brief			:strptime - Format string as time
+* @Scenario			:strptime - converts time string as the time described in timeptr
+*					 structure as per the format specifiers.
+* API's covered		:strptime
```

Comment:
Please check indentation and remove unnecessary gaps.

Reply:
modify